tests/cupyx_tests/scipy_tests/signal_tests/test_signaltools.py

```diff
@@ -213,10 +213,12 @@ def test_correlate2d(self, xp, scp, dtype):
 class TestConvolve2DEdgeCase:
 
     @testing.numpy_cupy_allclose(atol=1e-5, rtol=1e-5, scipy_name='scp')
+    @testing.with_requires('scipy>=1.10')
     def test_convolve2d_1(self, xp, scp):
         # see cupy/cupy#5989
         from scipy import misc
-        ascent = misc.ascent()
+        with pytest.warns(DeprecationWarning):
+            ascent = misc.ascent()
```

Comment:
`scipy.misc` will be removed in SciPy version 1.12.0. How about using `scipy.datasets.ascent` instead?

```suggestion
        if np.lib.NumpyVersion(scipy.__version__) >= '1.10.0':
            ascent = scipy.datasets.ascent()
        else:
            ascent = scipy.misc.ascent()
```

Reply:
Thanks!